extensions/amp-carousel/0.1/base-slide.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {timer} from '../../../src/timer';
+import {BaseCarousel} from './base-carousel';
+
+export class BaseSlide extends BaseCarousel {
```

Comment:
if we have a class called 'BaseCarousel' - per the same logic - it should be 'BaseSlide' right - why should it be plural?